### DISCUSSION TITLE
Is right-click not working on brush intentional?

### DISCUSSION THREAD

**Discussion by ziplantil** [PintaProject/Pinta#754] [Q&A] [answered]
Hi, I recently built from ac2588c636473f94a86fe4ef75340e37f2a67811 (native Linux build), and right-clicking to draw with the secondary color only works for the pencil tool, not the brush tool. Is this intentional (annoying if so) or a bug?

**Comment by cameronwhite** [comment]
I think that's definitely a bug, thanks for noticing!

**Comment by cameronwhite** [comment, marked as answer]
This is fixed in e01d0a23be2e6d6729810e626205f81e22058873
